src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ElasticJobExecutionCrudTests.ps1

```diff
@@ -0,0 +1,339 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+	.SYNOPSIS
+	Tests starting a job with default parameters
+#>
+function Test-StartJob
+{
+	# Setup
+	$a1 = Create-ElasticJobAgentTestEnvironment
+
+	# Setup SELECT 1 script on control db
+	$script = "SELECT 1"
+	$s1 = Get-AzureRmSqlServer -ResourceGroupName $a1.ResourceGroupName -ServerName $a1.ServerName
+	$serverLogin = $s1.SqlAdministratorLogin
+	$serverPassword = "t357ingP@s5w0rd!"
+	$credential = new-object System.Management.Automation.PSCredential($serverLogin, ($serverPassword | ConvertTo-SecureString -asPlainText -Force))
+	$jc1 = $a1 | New-AzureRmSqlElasticJobCredential -Name (Get-UserName) -Credential $credential
+	$tg1 = $a1 | New-AzureRmSqlElasticJobTargetGroup -Name (Get-TargetGroupName)
+	$tg1 | Add-AzureRmSqlElasticJobTarget -ServerName $a1.ServerName -DatabaseName $a1.DatabaseName
+	$j1 = Create-JobForTest $a1
+	$js1 = Create-JobStepForTest $j1 $tg1 $jc1 $script
+
+	try
+	{
+		# Start job
+		$je = Start-AzureRmSqlElasticJob -ResourceGroupName $j1.ResourceGroupName -ServerName $j1.ServerName -AgentName $j1.AgentName -JobName $j1.JobName
```

Comment:
When we run start-job multiple times in a row it will create new running executions each time.
Running stop multiple times on an execution will if execution is active, will start a cancellation attempt. Further stops on same execution with do nothing since first cancellation was in effect.